src/pages/User/components/PullRequests/index.js

```diff
@@ -8,6 +8,39 @@ import PullRequest from './PullRequest';
 import IssuesLink from './IssuesLink';
 import MeLinkInfo from './MeLinkInfo';
 import { GITHUB_TOKEN } from '../../../../config';
+import NotAMember from '../Modal/NotAMember';
+
+const GITHUB_ORG_NAME = 'leapfrogtechnology';
+
+/**
+ * returns an object containing user info
+ *
+ * @param {string} username
+ * @returns {*}
+ */
+export async function fetchUserInfo(username) {
+  const apiUrl = [
```

Comment:
It's a list of urls (plural) - the name should reflect that too. 
```suggestion
  const apiUrls = [
```